README.md

```diff
@@ -8,39 +8,29 @@
 
-An orchestration platform for the development, production, and observation of data assets.
-
-Dagster lets you define jobs in terms of the data flow between reusable, logical components, then test locally and run anywhere. With a unified view of jobs and the assets they produce, Dagster can schedule and orchestrate Pandas, Spark, SQL, or anything else that Python can invoke.
-
-Dagster is designed for data platform engineers, data engineers, and full-stack data scientists. Building a data platform with Dagster makes your stakeholders more independent and your systems more robust. Developing data pipelines with Dagster makes testing easier and deploying faster.
-
-### Develop and test locally, then deploy anywhere
-
-With Dagster’s pluggable execution, the same computations can run in-process against your local file system, or on a distributed work queue against your production data lake. You can set up Dagster’s web interface in a minute on your laptop, deploy it on-premise, or in any cloud.
-
-### Model and type the data produced and consumed by each step
-
-Dagster models data dependencies between steps in your orchestration graph and handles passing data between them. Optional typing on inputs and outputs helps catch bugs early.
-
-### Link data to computations
-
-Dagster’s Asset Manager tracks the data sets and ML models produced by your jobs, so you can understand how they were generated and trace issues when they don’t look how you expect.
-
-### Build a self-service data platform
-
-Dagster helps platform teams build systems for data practitioners. Jobs are built from shared, reusable, configurable data processing and infrastructure components. Dagit, Dagster’s web interface, lets anyone inspect these objects and discover how to use them.
-
-### Avoid dependency nightmares
-
-Dagster’s repository model lets you isolate codebases so that problems in one job don’t bring down the rest. Each job can have its own package dependencies and Python version. Jobs are run in isolated processes so user code issues can't bring the system down.
-
-### Debug pipelines from a rich UI
-
-Dagit, Dagster’s web interface, includes expansive facilities for understanding the jobs it orchestrates. When inspecting a run of your job, you can query over logs, discover the most time consuming tasks via a Gantt chart, re-execute subsets of steps, and more.
-
-## Getting Started
-
-### Installation
+Dagster is an orchestration platform for the development, production, and observation of data assets.
+
+- **Develop and Test Locally, then Deploy Anywhere:** With Dagster, the same computations can run
+in-process against your local file system, or on a distributed work queue against your production
+data lake. You can choose to locally develop on your labtop, deploy on-premise, or run in any
```

Comment:
```suggestion
data lake. Choose to locally develop on your laptop, deploy on-premise, or run in any
```

I vote for using an understood subject for conciseness. It also makes this sentence sound more compelling.